src/main/java/com/device/shop/controller/ProductController.java

```diff
@@ -17,13 +20,50 @@ public class ProductController {
 
     ProductService productService;
 
+    @PostMapping("/update/product/{id}")
```

Comment:
Change all urls in this controller (ALL methods) so they will start from `product`